### DISCUSSION TITLE
Some issues that arise during use

### DISCUSSION THREAD

**Discussion by learnsui1** [nomi-san/parsec-vdd#51] [Q&A]
Hello, I am using a Windows 11 laptop. When using Parsecvdisplay, it was successfully created and can be used normally on my other device. However, when I clicked a button for the Windows 11 projection function, its name here is: Second Screen Only. After clicking, my computer started to black out and the receiving device did not respond. This has caused my own computer to be unable to function properly. I have to restart and shut down the computer, and uninstalling and reinstalling ParsecVDisplay cannot solve this problem. I hope there is a solution. Thank you very much

**Comment by nomi-san** [comment]
That's normal behaviour when you don't have any external physical monitor. Why do you need to use this function "Second screen only"? If using with a projector, make sure you have duplicated the projection screen.
